packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -231,6 +232,16 @@ abstract class MultiDragGestureRecognizer extends GestureRecognizer {
   /// [Drag] object returned by this callback.
   GestureMultiDragStartCallback? onStart;
 
+  /// Called when interaction starts. This limits the dragging behavior
+  /// for custom clicks (such as scroll click).
+  /// When value is `(int buttons) => false`, allow no interactions.
+  /// When value is `(int buttons) => true`, allow all interactions.
+  /// When value is `(int buttons) => buttons & kPrimaryButton != 0`, allow
+  /// if selection contains a primary button.
+  ///
+  /// Defaults to `(int buttons) => buttons & kPrimaryButton != 0`.
```

Comment:
I think the default value should be explained rather than showing the code, especially since it involves bitwise operations.

Reply:
Thanks. I made the default `buttons == kPrimaryButton` to simplify things (the other reviewer agrees). I'll write what's possible to do with bitwise.